analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -378,8 +386,26 @@ def construct_config_handler(cls, args, context):
         checkers = ClangSA.get_analyzer_checkers(handler, environ)
 
         # Read clang-sa checkers from the config file.
-        clang_sa_checkers = context.checker_config.get(cls.ANALYZER_NAME +
-                                                       '_checkers')
+        clang_sa_configs = context.checker_config.get(cls.ANALYZER_NAME)
+
+        clangsa_version = ClangSA.get_analyzer_version_string(handler, environ)
+        matching_config = next(iter(filter(
+            lambda x: re.match(x["version_regex"], clangsa_version),
+            clang_sa_configs,
+        )), None)
+
+        clang_sa_profiles = None
+
+        version_to_log = clangsa_version.splitlines()[0]
+
+        if not matching_config:
+            LOG.debug(
+                "No matching configuration found for version:'{}'!".format(
+                    version_to_log))
```

Comment:
You can use placeholders instead instead of format:
```py
LOG.debug("No matching configuration found for version:'%s'!", version_to_log)
```

Reply:
Fixed.